composer.json

```diff
@@ -4,6 +4,9 @@
         "ext-gd": "0",
         "ext-curl": "0"
     },
+    "suggest": {
+        "ext-imagick": "to use the Imagick implementation",
```

Comment:
@mlebkowski it is, you can use this feature or not... and what's more i don't want to install this shit on my private server and i want to have possibility to fetch other fixes from this repo :smiling_imp: